frontend/src/PlayersView/PlayersView.jsx

```diff
@@ -1,28 +1,80 @@
-import StatsTable from "./StatsTable";
+import StatsTable from "../TableComponents/StatsTable";
+import SortMenu from "../TableComponents/SortMenu";
 import Header from "../Header/Header";
+import SearchBar from "../TableComponents/SearchBar";
 import { useState, useEffect } from "react";
+import TablePagination from "@mui/material/TablePagination";
 
 function PlayersView() {
   const [playersList, setPlayersList] = useState([]);
+  const [playersDisplayed, setPlayersDisplayed] = useState([]);
   const [page, setPage] = useState(1);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const [sortType, setSortType] = useState("");
+  const [searchQuery, setSearchQuery] = useState("");
 
-  async function loadPlayers(page) {
-    const response = await fetch(
-      `https://nba-stats-db.herokuapp.com/api/playerdata/?page=${page}`
-    );
-    const data = await response.json();
-    const players = data.results;
+  const handleChangePage = (event, newPage) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(parseInt(event.target.value, 10));
+    setPage(0);
+  };
+
+  async function loadPlayers(sortType) {
+    let queryUrl = new URL("http://localhost:5000/players");
+    queryUrl.searchParams.append("sortType", sortType);
+    const response = await fetch(queryUrl);
```

Comment:
I think you don't need the domain (http://localhost:5000) part of this url. fetch will handle relative to the current one I think?